src/frontend-build.sh

```diff
@@ -21,6 +21,7 @@ trap "teardown_docker" EXIT SIGINT SIGTERM
 if echo $JOB_NAME | grep -w "pr-check" > /dev/null; then
   CONTAINER_NAME="$APP_NAME-pr-check-$ghprbPullId"
   IS_PR=true
+  export IMAGE_TAG="pr-${ghprbPullId}-${IMAGE_TAG}"
```

Comment:
This looks good to me. There was something on the bonfire side we had to do for backends. We should checkup with squizz and see if that's already handled or not.

Reply:
Are all frontends hosted in github or are some in gitlab? You might need logic like this:

https://github.com/RedHatInsights/bonfire/blob/master/cicd/bootstrap.sh#L44-L51

But -- do the frontend jobs not already `source bootstrap.sh`?